SU-SWS/linky_clicky
# Update for changes in SoE

# NOT READY FOR REVIEW

# Summary
- We removed the credits field for images on SoE. This updates the Beans so that they no longer test for the credits fields. See ticket: https://stanfordits.atlassian.net/browse/SOE-3065
This will probably fail on image issues. I have a separate ticket to fix that:
https://stanfordits.atlassian.net/browse/SOE-3066

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch: 5.x-SOE-3065-credits
- Navigate to linky_clicky/sites/engineering
- Run behat test:
 `../../bin/behat -p sites-jse-soe-stage -s dev features/engineering_stanford_bean_types`
- Verify it _doesn't_ fail with any credit checks. 
It may fail on image issues. I have a separate ticket to fix that:
https://stanfordits.atlassian.net/browse/SOE-3066

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3065
Collections: https://stanfordits.atlassian.net/browse/SOE-3099

## Related PRs

## More Information

## Folks to notify



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
